Makefile.am

```diff
@@ -56,7 +31,40 @@ service_DATA = \
 	search-provider/com.endlessm.EknServices.SearchProviderV1.service \
 	$(NULL)
 
+search-provider/eks-search-provider-dbus.h search-provider/eks-search-provider-dbus.c: search-provider/eks-search-provider-dbus.xml Makefile.am
+	$(AM_V_GEN) $(GDBUS_CODEGEN) \
+	--interface-prefix=org.gnome.Shell \
+	--c-namespace Eks \
+	--generate-c-code search-provider/eks-search-provider-dbus \
+	$<
+
+search-provider/eks-knowledge-app-dbus.h search-provider/eks-knowledge-app-dbus.c: search-provider/eks-knowledge-app-dbus.xml Makefile.am
+	$(AM_V_GEN) $(GDBUS_CODEGEN) \
+	--interface-prefix=com.endlessm. \
+	--c-namespace Eks \
+	--generate-c-code search-provider/eks-knowledge-app-dbus \
+	$<
+
```

Comment:
Also, when I build from a fresh checkout with `make -j1` I get this:
```
  GEN      search-provider/eks-knowledge-app-dbus.c
Traceback (most recent call last):
  File "/usr/bin/gdbus-codegen", line 39, in <module>
    sys.exit(codegen_main.codegen_main())
  File "/usr/share/glib-2.0/codegen/codegen_main.py", line 188, in codegen_main
    h = open(c_code + '.h', 'w')
FileNotFoundError: [Errno 2] No such file or directory: 'search-provider/eks-knowledge-app-dbus.h'
Makefile:926: recipe for target 'search-provider/eks-knowledge-app-dbus.c' failed
```
Think you're missing a `$(srcdir)` here somewhere.
And when I have more than one parallel job it still goes on to compile the C code that includes the generated .h files, even though generating them failed.